internal/dbtest/db_test.go

```diff
@@ -879,7 +879,7 @@ func testWithForeignKeys(t *testing.T, db *bun.DB) {
 		ID       int `bun:",pk,autoincrement"`
 		UserID   int
 		UserType string
-		User     *User `bun:"rel:belongs-to,join:user_id=id,join:user_type=type"`
```

Comment:
It'd still be good to test behavior without the new tags. Replacing this with additional tests as discussed